content/references/rippled-api/admin-rippled-methods/admin-rippled-methods.md

```diff
@@ -7,14 +7,14 @@ Admin methods are meant only for trusted personnel in charge of keeping the serv
 Admin commands are available only if you connect to `rippled` on a host and port that the `rippled.cfg` file identifies as admin. By default, the commandline client uses an admin connection.
```

Comment:
Suggestion: We can remove this file entirely and use the landing to generate it automatically. Make this first line the "blurb" of the "Admin rippled Methods" page.